analytics/sprint-oac-auto-insights-pt/sprint-oac-auto-insights-pt.md

```diff
@@ -44,4 +44,4 @@ Oracle Analytics irá disponibilizar sugestões de visualizações com uma descr
 ## Reconhecimentos
 * **Autor** - Lucian Dinescu, Product Strategy, Analytics
 * **Tradução** - Thais Henrique, Cloud Engineer, Brazil Data & AI Team
-* **Última Atualização** - Thais Henrique,  Agosto 2022
+* **Última Atualização** - sprint-oac-add-notes-pt
```

Comment:
@ThaisHenrique Could you update the **Última Atualização** to have the correct name and date

Reply:
Yes, it is done